.gitlab-ci.yml

```diff
@@ -31,11 +31,13 @@ build-production:
   extends: .build
   variables:
     DOCKER_IMAGE_NAME: "$CI_PROJECT_NAME-production"
-    DOCKER_BUILD_ARG_REACT_APP_OIDC_AUTHORITY: "https://tunnistamo.test.kuva.hel.ninja/"
-    DOCKER_BUILD_ARG_REACT_APP_PROFILE_GRAPHQL: "https://helsinkiprofile.test.kuva.hel.ninja/graphql/"
+    DOCKER_BUILD_ARG_REACT_APP_OIDC_AUTHORITY: "https://api.hel.fi/sso/openid"
+    DOCKER_BUILD_ARG_REACT_APP_PROFILE_GRAPHQL: "https://helsinkiprofile.prod.kuva.hel.ninja/graphql/"
```

Comment:
This value should probably be "https://helsinkiprofile.hel.fi/graphql/" since that's the public domain name for the backend.

Reply:
Thanks! Looks like that url is already resolving correctly.